kured-ds.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     matchLabels:
       name: kured
   updateStrategy:
-   type: RollingUpdate
+    type: RollingUpdate
```

Comment:
while these whitespace fixes *are* readability improvements, let's omit them from this PR as they aren't related

Reply:
allright